README.md

```diff
@@ -149,7 +149,7 @@ By default, when building the project with no options, the following static libr
   - az_storage_blobs
     - Storage SDK blobs client.
   - az_noplatform
-    - A platform abstraction which will compile but returns `AZ_ERROR_NOT_IMPLEMENTED` for all platform calls. This ensures the project can be compiled without the need to provide any specific platform implementation. This is useful if you want to use az_core without platform specific functions like `mutex` or `time`.
+    - A platform abstraction which will compile but returns 0 or does nothing for all platform calls. This ensures the project can be compiled without the need to provide any specific platform implementation. This is useful if you want to use az_core without platform specific functions like `time` or `sleep`.
   - az_nohttp

```

Comment:
Mention that this will return `AZ_ERROR_DEPENDENCY_NOT_PROVIDED` at runtime.

Reply:
And maybe it should, let's get what @JeffreyRichter thinks, and complete that in a separate PR.
Jeffrey, do you think we should change PAL functions (we only have two - `sleep()` and `clock()`) to return AZ_RESULT, and the `clock()` would return result as an out parameter? This way, if you ever seriously rely on a no-op PAL, the error will get to you, the stuff will start failing earlier, with better error code.

Reply:
I am talking about `az_nohttp`, no the PAL.

Reply:
But this section talks about platform, not HTTP. It used to say it would return `AZ_ERROR_NOT_IMPLEMENTED` but reality is that it won't. So I touched the comment, but instead of renaming, I updated it to say what it actually does, which is to return 0 or do nothing.

Reply:
README.md line 25 talks about `az_nohttp`, and I updated it with DEPENDENCY_NOT_PROVIDED.

Reply:
https://github.com/Azure/azure-sdk-for-c/pull/1219